gorush/notification_apns.go

```diff
@@ -180,9 +200,16 @@ Retry:
 
 	for _, token := range req.Tokens {
 		notification.DeviceToken = token
-
 		// send ios notification
-		res, err := ApnsClient.Push(notification)
+		var err error
+		var res *apns.Response
+		if PushConf.Ios.Development {
+			res, err = DevApnsClient.Push(notification)
+		}
+		if (err != nil || res.StatusCode != 200) && PushConf.Ios.Production {
+			// Try Production second, so that if it errors out, these are the ones shown
```

Comment:
Why do we need to try both dev and production in same run?

Reply:
Team wants to be able to test both Dev Builds and Prod builds on staging first, and since we can't identify if token is prod or dev easily, can just try both if both are activated